dev-docs/bidders/ix-server.md

```diff
@@ -66,12 +68,73 @@ The following table lists the media types that Index supports. For information a
 | video   | Supported, including ad pods for OTT    |
 | native      | Supported       |
 
+<a id="hosting-instance"></a>
+
+## Hosting your own Prebid Server instance 
+
+**Before you begin:** Contact your Index Exchange Representative to get an endpoint and setup instructions.
+
+If you are hosting your own Prebid Server instance, depending on whether you are using Prebid Server Go or Prebid Server Java version, complete one of the following steps: 
+
+* If you are using the Prebid Server Go version, in the `static/bidder-info/ix.yaml` file, complete the following:
+
+  * Enable the adapter by deleting the `disabled: true` entry.
+  * Add the following new entry and include the regional endpoint provided to you by Index:
+
+    ```javascript
+     endpoint: "https://<ENDPOINT URL>"
+     ```
+
+  * Edit the below existing entry and include your publisher ID in the `s` parameter:
+
+    ```javascript
+     userSync:  
+      redirect:  
+       url: "https://ssum.casalemedia.com/usermatchredir?s=<PUBLISHER ID>&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
+     ```
+
+  * Edit the below existing entry and include your publisher ID in the `s` parameter:
+
+     ```javascript
+     userSync:  
+      redirect:  
+       iframe: "https://ssum.casalemedia.com/usermatch?s=<PUBLISHER ID>&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
+      ```         
+
+* If you are using [Prebid Server Java](https://github.com/prebid/prebid-server-java) version, edit the `prebid-server-java` entry in the `src/main/resources/bidder-config/ix.yaml` file as follows:
+
+  * Edit the below existing entry and include the endpoint URL provided to you by Index.
+
+    ```javascript
+    adapters: 
+     ix: 
+       endpoint: "https://<ENDPOINT URL>"
+    ```
+
+  * Edit the below existing entry and include your publisher ID in the `s` parameter:
+
+    ```javascript
+     adapters: 
+       ix: 
+        usersync: 
+         redirect: 
+          url: "https://ssum.casalemedia.com/usermatchredir?s=<PUBLISHER ID>&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}" 
+    ```
+
+  * Add the below entry and include your publisher ID in the `s` parameter:
+
+    ```javascript
+    adapters: 
+      ix: 
+       usersync: 
+        iframe: 
+         url: "https://ssum.casalemedia.com/usermatch?s=<PUBLISHER ID>&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
+    ```
+
 <a id="call-index"></a>
```

Comment:
```suggestion
```

This is not necessary. headings provide unique ids already

Reply:
we have a TOC at the top that uses these IDs. Do you think those will break if I remove them?